src/lib/ReturnTop/ReturnTop.svelte

```diff
@@ -0,0 +1,46 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  const dispatch = createEventDispatcher();
+  import "../style/app.css";
+  const arrow = `<svg width="30" height="30" viewBox="0 0 40 40" xmlns="http://www.w3.org/2000/svg">
+<rect width="40" height="40" fill="#fff"/>
+<circle cx="20" cy="20" r="17"/>
+<path d="m20.264 8.4834 13.052 12.918h-26.103l13.051-12.918z" fill="#fff"/>
+<rect x="15.241" y="15.659" width="10.047" height="14.353" fill="#fff"/>
+</svg>`;
```

Comment:
First off, to answer your original question, I think it would be totally fine when necessary to include SVG assets like this that are only used in 1 component inside the component's folder. That is part of the idea of each component living inside its own directory, any supporting assets and stuff can all be stored close by. But I agree that in this case, the arrow is simple enough that it doesn't need its own file. 

Going one step further, it might make more sense/be a little cleaner to just put this markup directly in the HTML are below the span rather than making it a string variable and including it with the `{@html arrow}` bit?